training_model/grpo_train.py

```
@@ -22,12 +22,15 @@ def reward_function(completions: list, row: dict) -> list:
     Returns:
         list: List of reward values for each completion
     """
-    correct_actions = row["correct_actions"]
+    row = kwargs.get("row", None)
```

Ruff (0.8.2)

Use `kwargs.get("row")` instead of `kwargs.get("row", None)`

Replace `kwargs.get("row", None)` with `kwargs.get("row")`

(SIM910)